.github/workflows/check-modified-assets.yml

```diff
@@ -10,15 +10,28 @@ jobs:
             - uses: actions/checkout@v3
               with:
                   ref: trunk
+            - name: Cache node_modules
+              id: cache-node-modules
+              uses: actions/cache@v3
+              env:
+                  cache-name: cache-node-modules
+              with:
+                  path: node_modules
+                  key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+
             - name: Setup node version and npm cache
               uses: actions/setup-node@v3
               with:
                   node-version-file: '.nvmrc'
                   cache: 'npm'
```

Comment:
Please verify if we still need this here and if this is not actually doing the same behind the scene. Since setup-node@v3 (or maybe even v2?) caching with sane default handling was handled by the `actions/setup-node`

Reply:
Yep, thanks for pointing that out, that was my first thought too. But it doesn't cache node_modules, only the .npm cache. Which is still useful when we do have to install dependencies, it avoids downloading whatever it has in the cache. It states clearly in the docs

> Note: The action does not cache node_modules

So I think it's worth having both. 

It

Reply:
Yes, they do slightly different things. Cachine node_modules is useful for not having to do npm ci at all. But when we change a package version, we will need to npm ci. This is where the npm cache is useful, as it will grab whatever packages haven't changed from a local cache, and it will only download the ones that have changed, saving bandwidth. So they work together well

Reply:
Thank you for the explanation, @alexflorisca!